.gitignore

```diff
@@ -1 +1,3 @@
-target
\ No newline at end of file
+target
+.idea
```

Comment:
usually I'd keep them in the user's`.gitignore` file (otherwise there'd be too many excludes for too many IDEs).